sklearn/metrics/_classification.py

```diff
@@ -1348,6 +1348,12 @@ def fbeta_score(
     averaged depending on the `average` parameter. Use `labels` specify the set of
     labels to calculate F-beta score for.
 
+    F-beta score is not implemented as a named scorer that can be
```

Comment:
I'm wondering if we want to have this is the long summary or instead to have it in the "Notes" section below. I find the current section already quite long and here this is more of an implementation detail and we could keep the long summary related to the theoretical/mathematical story telling.

@marijavlajic @ogrisel WDYT?

Reply:
I think it's good to add sentence and refer to the examples for usage details.

Reply:
I am not sure I completely understand what you mean by adding a sentence and referring to the examples, but I've added a more detailed example and moved the part about using make_scorer to the Notes section as @glemaitre suggested.